typedb/connection/client.py

```diff
@@ -18,7 +18,11 @@
 #   specific language governing permissions and limitations
```

Comment:
Instead of launching a background thread in each session to send pulses, we now use a single thread in the Client to send pulses for all of its sessions. This allows the client to handle a much larger number of sessions than it previously could.